apigentools/commands/init.py

```diff
@@ -19,7 +19,6 @@ class InitCommand(Command):
     CONFIG_FILE_JSON = {
         "codegen_exec": "openapi-generator",
         "languages": {},
-        "server_base_urls": {"v1": "https://api.myserver.com/v1"},
```

Comment:
This omission should also be accompanied by adding the `server` key to `V1_HEADER_JSON`.

Reply:
done